example/test/scene_test_extensions.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:stager/stager.dart';
+
+/// Convenience functions for widget testing [Scene]s.
+///
+/// TODO: move this to a new stager_testing package
+extension Testing on Scene {
```

Comment:
Should this be in a "Testing" folder?

Reply:
Because it has a dependency on flutter_test, it will likely need to be in a separate package (sort of like drift and drift_dev). I'm going to file a ticket to do that, but I think it's going to be a decent amount of work.